docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -0,0 +1,229 @@
+[[url-drilldown]]
+=== URL Drilldown
+
+The URL drilldown allows you to navigate from a dashboard to an internal or external URL.
+The destination URL can be dynamic, depending on the dashboard context or user’s interaction with a visualization.
+
+For example, you might have a dashboard that shows data from a Github repository.
+You can create a drilldown that navigates from this dashboard to Github.
+Refer to the <<try-it, Try it section>> for instructions on how to create this drilldown.
+
+[role="screenshot"]
+image:images/url_drilldown_go_to_github.gif[Drilldown on pie chart that navigates to Github]
+
+NOTE: URL drilldown is available with the https://www.elastic.co/subscriptions[Gold subscription] and higher.
+
+[float]
+[[try-it]]
+==== Try it: Create a URL drilldown
+
+This example shows how to create the "Show on Github" drilldown shown earlier in this doc.
+
+. Add the <<gs-get-data-into-kibana, sample web logs>> data set.
+. Open the `[Logs] Web traffic` dashboard. This isn’t data from Github, but it should work for demonstration purposes.
+. In the dashboard menu bar, click *Edit*.
+. In `[Logs] Visitors by OS`, open the panel menu, and then select *Create drilldown*.
+. Give the drilldown a name: *Show on Github*.
+. Select a drilldown action: *Go to URL*.
+[role="screenshot"]
+image:images/url_drilldown_pick_an_action.png[Action picker]
+. Enter a URL template: 
++
+```
+https://github.com/elastic/kibana/issues?q=is:issue+is:open+{{event.value}}
```

Comment:
This code block looks like so:

![image](https://user-images.githubusercontent.com/9773803/92451370-e668e080-f1bc-11ea-9a36-9e3e734a5088.png)

Maybe we can make it like this:

![image](https://user-images.githubusercontent.com/9773803/92451404-eec11b80-f1bc-11ea-9f12-dfedb4818b10.png)

Reply:
Thanks, agree